tools/gen_sig_info.py

```diff
@@ -229,13 +235,12 @@ def extract_sigs(symbols, obj_file):
 
 
 def extract_sig_info(sig_info, extra_settings=None, extra_cflags=None):
+  print(' .. ' + str(extra_settings) + ' + ' + str(extra_cflags))
```

Comment:
temporary debug logging?

Reply:
This is useful logging that only folks who run this tool will see (i.e. folks working on emscripten itself).